source/jormungandr/jormungandr/instance.py

```diff
@@ -366,6 +366,31 @@ def realtime_pool_size(self):
         instance_db = self.get_models()
         return get_value_or_default('realtime_pool_size', instance_db, self.name)
 
+    @property
+    def min_nb_journeys(self):
```

Comment:
if we add `min_nb_journeys` we should also add `max_nb_journeys`

Reply:
@kinnou02 What do you propose de default values for max_nb_journeys and max_successive_physical_mode?

Reply:
I will put them at `None`, there is also `float("inf")` but is will be harder to store in database.
you can mix both by storing null in the database and returning `float("inf")` in that case

Reply:
done